pkg/minikube/translate/translate.go

```diff
@@ -77,13 +77,13 @@ func DetermineLocale() {
 	translationFile := "pkg/minikube/translate/translations/" + preferredLanguage.String() + ".json"
 	t, err := ioutil.ReadFile(translationFile)
 	if err != nil {
-		glog.Errorf("Failed to load transalation file for %s: %s", preferredLanguage.String(), err)
+		glog.Infof("Failed to load translation file for %s: %v", preferredLanguage.String(), err)
```

Comment:
I may be incorrect, but I think the call to String() is likely unnecessary,  as %s should call it automatically.

Reply:
You're probably right yeah.